manifests/railsapp.pp

```diff
@@ -3,6 +3,7 @@
   include profile::base
   include profile::railsapp
   include profile::monit
+  include profile::datadog
```

Comment:
Add a conditional to this class somewhere for this.  I think the host_railsapp is best place but it's probably ok for it to be in profile as well.  Default should be false so we don't enable this everywhere.